src/NodaTime/TimeZones/IO/TzdbStreamData.cs

```diff
@@ -63,12 +62,6 @@ internal sealed class TzdbStreamData
         /// </summary>
         public IList<TzdbZone1970Location> Zone1970Locations { get; }
 
-        /// <summary>
-        /// Additional mappings from Windows standard name to TZDB ID. Primarily used in
-        /// the PCL build, where we can't get at the system ID. This never returns null.
-        /// </summary>
-        public IDictionary<string, string> WindowsAdditionalStandardNameToIdMapping { get; }
```

Comment:
Update `docs/developer/tzdb-file-format.md` to note that we're no longer using this field's data?